hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/Router.java

```diff
@@ -876,4 +878,10 @@ public void setConf(Configuration conf) {
     this.conf = conf;
   }
 
+  @VisibleForTesting
+  public boolean isAsync() {
+    return getRpcServer() != null ? getRpcServer().isAsync() :
+        getConfig() != null ? getConfig().getBoolean(DFS_ROUTER_ASYNC_RPC_ENABLE_KEY,
+            DFS_ROUTER_ASYNC_RPC_ENABLE_DEFAULT) : DFS_ROUTER_ASYNC_RPC_ENABLE_DEFAULT;
+  }
```

Comment:
Hi @hfutatzhanghb Can this method be deleted?

Reply:
@KeeProMise Nice advise, have deleted it.